File: Python/Easy/جمع الأعداد الزوجية في مصفوفة.py

# Creator: OmarAfet
# https://profile.satr.codes/OmarAfet/public/overview
# https://github.com/OmarAfet

def sum_even(arr: list[int]) -> int:
    x = []
    for i in arr:
        if i % 2 == 0:
            x.append(i)
    return sum(x)

File: JavaScript/Easy/جمع الأعداد الزوجية في مصفوفة.js
/*

Creator: OmarAfet
https://profile.satr.codes/OmarAfet/public/overview
https://github.com/OmarAfet

*/

function sum_even(arr) {
	x = 0;
	for (let i = 0; i < arr.length; i++) {
		const e = arr[i];
		if (e % 2 == 0) {
			x += e;
		}
	}
	return x;
}
